protocol/avdl/keybase1/account.avdl

```diff
@@ -76,4 +76,22 @@ protocol account {
    */
   void cancelReset(int sessionID);
   void timeTravelReset(int sessionID, string username, gregor1.DurationSec duration);
+
+  record TeamContactSettings {
+    TeamID teamID;
```

Comment:
use `@jsonkey("team_id")` here (and other places) so that the AVDL structs can map onto the API keynames

Reply:
why do so many fields in other avdl records not have jsonkey annotations?

Reply:
Most don't correspond 1-to-1 to API endpoints as these do.